resources/deb/etc/default/openhab2

```diff
@@ -1 +1,12 @@
+# openHAB 2 service configuration
+# 
+# Configure additional options for the JAVA_OPTS environment variable.
+# These will be appended to the execution of the openHAB Java runtime in front of all other options.
+# 
+# EXTRA_JAVA_OPTS="-Dgnu.io.rxtx.SerialPorts=/dev/ttyUSB0:/dev/ttyS0:/dev/ttyS2:/dev/ttyACM0:/dev/ttyAMA0"
+# EXTRA_JAVA_OPTS="-Djna.library.path=/lib/arm-linux-gnueabihf/ -Duser.timezone=Europe/Berlin -Dgnu.io.rxtx.SerialPorts=/dev/ttyS0"
```

Comment:
You think? I saw a similar example in another file in the folder but sure, will make it clearer.